executable/Env.re

```diff
@@ -1,5 +1,55 @@
 open Fnm;
 
+module ExportShellVarCmdGetters = {
+  type exportCmdGetter = {
+    getPathCmd: string => string,
+    getMultiShellPathCmd: string => string,
+    getFnmDirCmd: string => string,
+    getNodeDistMirrorCmd: string => string,
+    getLogLevelCmd: string => string,
+  };
+
+  let posixExportCmdGetter: exportCmdGetter = {
+    getPathCmd: path => Printf.sprintf("export PATH=%s/bin:$PATH", path),
+
+    getMultiShellPathCmd: path =>
+      Printf.sprintf("export %s=%s", Config.FNM_MULTISHELL_PATH.name, path),
+
+    getFnmDirCmd: fnmDir =>
+      Printf.sprintf("export %s=%s", Config.FNM_DIR.name, fnmDir),
+
+    getNodeDistMirrorCmd: nodeDistMirror =>
+      Printf.sprintf(
+        "export %s=%s",
+        Config.FNM_NODE_DIST_MIRROR.name,
+        nodeDistMirror,
+      ),
+
+    getLogLevelCmd: logLevel =>
+      Printf.sprintf("export %s=%s", Config.FNM_LOGLEVEL.name, logLevel),
+  };
+
+  let fishExportCmdGetter: exportCmdGetter = {
+    getPathCmd: path => Printf.sprintf("set -gx PATH %s/bin $PATH;", path),
+
+    getMultiShellPathCmd: path =>
+      Printf.sprintf("set -gx %s %s;", Config.FNM_MULTISHELL_PATH.name, path),
+
+    getFnmDirCmd: fnmDir =>
+      Printf.sprintf("set -gx %s %s;", Config.FNM_DIR.name, fnmDir),
+
+    getNodeDistMirrorCmd: nodeDistMirror =>
+      Printf.sprintf(
+        "set -gx %s %s",
+        Config.FNM_NODE_DIST_MIRROR.name,
+        nodeDistMirror,
+      ),
+
+    getLogLevelCmd: logLevel =>
+      Printf.sprintf("set -gx %s %s", Config.FNM_LOGLEVEL.name, logLevel),
+  };
+};
+
 let symlinkExists = path =>
   try%lwt (Lwt_unix.lstat(path) |> Lwt.map(_ => true)) {
```

Comment:
I didn't notice that you had merged the reformatted code, before I committed and pushed, so I had purposefully left in the space after `try%lwt` here.